src/Core/src/Handlers/ImageButton/ImageButtonHandler.cs

```diff
@@ -33,7 +33,7 @@ public partial class ImageButtonHandler : IImageButtonHandler
 			[nameof(IButtonStroke.StrokeColor)] = MapStrokeColor,
 			[nameof(IButtonStroke.CornerRadius)] = MapCornerRadius,
 			[nameof(IImageButton.Padding)] = MapPadding,
-#if WINDOWS
+#if ANDROID || WINDOWS
 			[nameof(IImageButton.Background)] = MapBackground,
 #endif
```

Comment:
What is the reason for this again on Android? I see the base view does all the same work:

![image](https://user-images.githubusercontent.com/1096616/174423405-902e4b9e-51b3-41bb-8453-79dd4b94da99.png)

Reply:
It's because we take the ImageMapper that already has a MapBackground that overrides this,